test/t8_test_cmesh_copy.c

```diff
@@ -24,8 +24,18 @@
 #include "t8_cmesh/t8_cmesh_trees.h"
 #include "t8_cmesh/t8_cmesh_partition.h"
 
+
+sc_MPI_Comm  comm_list[]={sc_MPI_COMM_WORLD,sc_MPI_COMM_NULL,sc_MPI_COMM_SELF};
+size_t num_comm = sizeof(comm_list) / sizeof(comm_list[0]); 
+int dim=3
+int min_dim=1
+int binary=2
+int max_num_of_trees=100
+int max_num_of_prisms=100
```

Comment:
We should replace these with preprocessor macros, i.e. 
```
#define T8_CMESH_TEST_DIMS 3
#define T8_CMESH_MIN_DIM 1
```
etc.

Reply:
Is there a reason why we start at dimension 1?
0-dimensional cmeshes are possible. The element class is T8_ECLASS_VERTEX